src/test/java/com/amadeus/jenkins/plugins/workflow/libs/HttpRetrieverIntegrationTests.java

```diff
@@ -225,14 +227,18 @@ FilePath buildJobWithLibrary(@Nullable String fixtureName, @Nullable String libr
                             .willReturn(WireMock.notFound()));
         }
         globalLibraries.getLibraries().add(new LibraryConfiguration(
-                libraryName,
-                new HttpRetriever(urlBuilder.apply(libraryName), credentials.getId(), withPreemptiveAuth)
+                "foo",
+                new HttpRetriever(urlBuilder.apply("foo"), credentials.getId(), withPreemptiveAuth)
         ));
         WorkflowJob p = j.jenkins.createProject(WorkflowJob.class, "p");
         p.setDefinition(new CpsFlowDefinition(importScript, true));
 
         WorkflowRun run = j.buildAndAssertSuccess(p);
-        return new FilePath(run.getRootDir().toPath().resolve("libs").resolve(libraryName).toFile());
+        List<Path> children = Files.list(run.getRootDir().toPath().resolve("libs"))
+                .filter(Files::isDirectory)
+                .collect(Collectors.toList());
+        assertThat(children, hasSize(1));
+        return children.get(0);
```

Comment:
Newer versions of `pipeline-groovy-lib` name this directory with a hash rather than the library name. Since it's the only child that is a directory, it's still easy to fish it out.